scripts/sanitycheck

```diff
@@ -3448,9 +3463,12 @@ def log_info_file(instance):
     h_log = "{}/handler.log".format(build_dir)
     b_log = "{}/build.log".format(build_dir)
     v_log = "{}/valgrind.log".format(build_dir)
+    d_log = "{}/device.log".format(build_dir)
 
     if os.path.exists(v_log) and "Valgrind" in instance.reason:
         log_info("{}".format(v_log))
+    elif os.path.exists(d_log):
+        log_info("{}".format(d_log))
     elif os.path.exists(h_log):
         log_info("{}".format(h_log))
     else:
```

Comment:
should we judge whether it's the flash error or tests failed via judging whether the h_log is empty?

if it's the test case failed, I think we would more like to see hander.log,  if it's the flash error, then it should be device.log.
for now, it's all device.log.